src/modules/ui/directives/qrCodeRead/QrCodeRead.js

```diff
@@ -185,7 +185,7 @@
             _parseQrCode({ result }) {
                 const [protocol, data] = result.split('://');
                 const [body, search] = data.split('?');
-                return { protocol, body, params: utils.parseSearchParams(search) };
+                return { protocol, body, params: utils.parseSearchParams(search || '') };
```

Comment:
It might be better to set default value inside `parseSearchParams` in case there is a need to always have `''` when an argument is missing.